style.css

```diff
@@ -0,0 +1,59 @@
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: row;
+}
+
+#welcome {
+  font-size: 20px;
+  justify-content: end;
+}
+
+.menu {
+  align-self: center;
+  padding-left: 280px;
+}
+
+.headline_section {
+  position: relative;
+  text-align: center;
+}
+
+.text {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.primary-text {
+  align-self: center;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+  text-justify: inter-word;
+}
+
+#second {
+  color: #36b37e;
+}
+
+.second-text {
+  width: 327px;
+  height: 164px;
+  font-weight: 400;
+  font-size: 20px;
+  align-items: center;
+  text-align: center;
+}
+
+.link {
+  align-content: center;
+  justify-content: space-around;
+  padding-left: 35%;
+}
```

Comment:
- Kindly make sure your icons are showing properly and the design of your icons looks like the Figma design as required in the project, follow this link to Check the [Figma design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)Check the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

- Check below your design compared to the Figma design :grinning: :grinning: :grinning: 

## Figma  design :+1: 

![image](https://user-images.githubusercontent.com/95297251/191819626-2770ea9b-91a3-43f8-8dd5-0cc02a6be7d5.png)

## Your design :-1: 

![image](https://user-images.githubusercontent.com/95297251/191821511-e05bd0a3-904c-46a4-8d1c-1b9701ecd21f.png)

Reply:
Designed as per Figma design. Please check.